src/widgets/splits/SplitContainer.hpp

```diff
@@ -21,6 +21,10 @@ namespace chatterino {
 class Split;
 class NotebookTab;
 class Notebook;
+struct ContainerNodeDescriptor;
+struct SplitNodeDescriptor;
+using NodeDescriptor =
+    std::variant<ContainerNodeDescriptor, SplitNodeDescriptor>;
```

Comment:
Questionable, I think we're best off just including WindowDescriptors.hpp here since the file is so small & this leaks the implementation a bit